### PR TITLE
Store pkg info in el8 containers

### PR DESCRIPTION
**Pull request by aandvalenzuela**

### CODE REVIEW

File: el8/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,8 @@ LABEL maintainer="CMS Build"
 LABEL name="CMS Worker Node on EL - Buildtime dependencies"
 
 # Dependencies on libidn libXmu libstdc++-static might be cleaned up in the future
-RUN dnf install --disablerepo=epel -y perl-Carp perl-PathTools perl-Data-Dumper perl-Errno perl-Exporter \
+RUN rpm -qa --queryformat '%{NAME} %{VERSION} %{RELEASE}\n' >> /tmp/versions-grid.tmp &&\
```
Comment (smuzaffar) [inline el8/Dockerfile:6]: does `rpm -qa` return sorted list? If not then better to add `| sort` for each `rpm -qa`

File: el8/Dockerfile [hunk 2]
```diff
@@ -26,6 +27,10 @@ RUN dnf --enablerepo=powertools install -y texinfo libstdc++-static &&\
 	dnf update -y ca-certificates &&\
 	dnf clean all &&\
 	/root/fix_ssh_config.sh /etc/ssh/ssh_config &&\
-	echo "Timestamp: @BUILD_DATE@" > /image-build-info.txt
+	echo "Timestamp: @BUILD_DATE@" > /image-build-info.txt &&\
+	rpm -qa --queryformat '%{NAME} %{VERSION} %{RELEASE}\n' >> /tmp/versions-grid.tmp &&\
```
[the remark this answers is not in the document]
Comment (aandvalenzuela) [reply el8/Dockerfile:31]: Thanks, it should be `/tmp/versions-buildtime.tmp`

**Comment by smuzaffar** [comment]
I would suggest to not have hard coded names and just use `package-base.txt` and `package-image.txt`

File: el8/Dockerfile (hunk 1 above)
Comment (aandvalenzuela) [reply el8/Dockerfile:6]: It is not sorted by default, so I will do `sort` for each `rpm -qa`

**Comment by smuzaffar** [comment]
every image is dumping package list at start and at the end. so just use `package-base.txt` and `package-image.txt`

**Comment by aandvalenzuela** [comment]
Thanks! I am opening a follow-up PR with the suggestions (I am testing if there is a big increase in the image size)

**Pull request merged**